src/test/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1Test.java

```diff
@@ -136,25 +136,64 @@ private Grouping grouping() {
         Grouping grouping = new Grouping("test:ing:me:bob");
 
         Group basisGroup = new Group();
-        basisGroup.addMember(new Person("b0-name", "b0-uuid", "b0-username"));
-        basisGroup.addMember(new Person("b1-name", "b1-uuid", "b1-username"));
-        basisGroup.addMember(new Person("b2-name", "b2-uuid", "b2-username"));
-        grouping.setBasis(basisGroup);
+        Subject subjectBasis0 = new Subject();
+        subjectBasis0.setUid("b0-uid");
+        subjectBasis0.setName("b0-name");
+        subjectBasis0.setUhUuid("b0-uuid");
```

Comment:
Maybe adding a constructor would be worth it here. Let me know if there's any difficulties with making the constructor. I know that the WsSubject class is hard to deal with.